### ISSUE TITLE
Classic Client Adds Unnecessary Punctuation After Channel and Text Message TTS Announcements

### ISSUE THREAD

**Issue by Simon818** [BearWare/TeamTalk5#316]
This should be an easy one. Whenever a channel or text message announcement is sent directly to the screen reader via the Tolk library, there is an extra period at the end of them. This is not necessary as screen readers typically pause between lines anyway, and it also creates confusion if the user decides to add their own punctuation (for instance, a sentence that ends in a question mark won't be spoken as a question because of the trailing period, and a sentence ending in a period might have a "dot dot" spoken at the end for the same reason). I recommend removing this punctuation from the string format entirely. The screen reader should take care of this on its own. Alternatively, you can check the last character of the string. if it's an upper or lower case letter or a number, add a period. If not, just send as is. I'm pretty confident that excluding it entirely and saving yourself the extra work would be fine though.

**Comment by bear101** [comment]
You can try to edit the English.lng file and then switch to that one in 
the Preferences -> Window tab. If you manage to improve the "speak time" 
then just email the updated English.lng file to me.

--
Kind regards

Bjoern D. Rasmussen
www.bearware.dk

**Comment by OzancanKaratas** [comment]
The request should be globalize, so we should edit the TeamTalk Classic source. If you want, I can do.

**Comment by OzancanKaratas** [comment]
@bear101 Please approve my request, I want to write code for TeamTalk. I sent a request to Contributors team.

**Comment by bear101** [comment]
It's better just to do pull-requests. Then there's less chance that something else breaks.

**Comment by bear101** [comment]
@Oreonan Is this issue still relevant?

**Comment by Oreonan** [comment]
@bear101 No, I've fixed it in new translations.
